pusher/src/Controller/AuthenticateController.ts

```diff
@@ -197,6 +199,13 @@ export class AuthenticateController extends BaseHttpController {
                             locale: authTokenData?.locale,
                         });
                     } catch (err) {
+                        if(Axios.isAxiosError(err)){
+                            const errorType = isErrorApiData.safeParse(err?.response?.data);
+                            if (errorType.success) {
+                                res.sendStatus(err?.response?.status ?? 500);
+                                return res.json(errorType.data);
+                            }
+                        }
```

Comment:
This part feels wrong.
If there is any exception, or an exception that is an Axios error BUT that is not parsable.... we do nothing except dumping a loosy message in the logs.
And the HTTP return is actually empty?